From a416b277040c721159ac45f257c8cd37588a0f52 Mon Sep 17 00:00:00 2001
From: Pino Toscano <ptoscano@redhat.com>
Date: Thu, 15 Jan 2015 10:56:53 -0500
Subject: [PATCH 118/118] RHEL 6: sysprep: remove /etc/shadow note

Once RHBZ#1160261 is in and we can require the augeas providing it,
virt-sysprep will remove users from /etc/shadow too; hence, remove
the documentation note (and the code comment) mentioning that issue.

Related to RHBZ#1138630.
---
 sysprep/sysprep_operation_user_account.ml | 8 --------
 1 file changed, 8 deletions(-)

diff --git a/sysprep/sysprep_operation_user_account.ml b/sysprep/sysprep_operation_user_account.ml
index b394296..ed4f7b9 100644
--- a/sysprep/sysprep_operation_user_account.ml
+++ b/sysprep/sysprep_operation_user_account.ml
@@ -49,10 +49,6 @@ let user_account_perform g root =
             | None ->
               eprintf "virt-sysprep: user-accounts: missing '/' in %s\n" userpath;
               exit 1 in
-          (* XXX Augeas doesn't yet have a lens for /etc/shadow, so the
-           * next line currently does nothing, but should start to
-           * work in a future version.
-           *)
           g#aug_rm (sprintf "/files/etc/shadow/%s" username);
           g#aug_rm (sprintf "/files/etc/group/%s" username);
           g#rm_rf ("/home/" ^ username)
@@ -71,10 +67,6 @@ let op = {
     pod_description = Some (s_"\
 Remove all the user accounts and their home directories.
 The \"root\" account is not removed.");
-    pod_notes = Some (s_"\
-Currently this does not remove the user accounts from
-C</etc/shadow>.  This is because there is no lens for
-the shadow password file in Augeas.");
     perform_on_filesystems = Some user_account_perform;
 }
 
-- 
2.7.4

